packages/components/src/snackbar/index.js

```diff
@@ -34,8 +35,7 @@ function Snackbar( {
 	const classes = classnames( className, 'components-snackbar' );
 	if ( actions && actions.length > 1 ) {
 		// we need to inform developers that snackbar only accepts 1 action
-		// eslint-disable-next-line no-console
-		console.warn( 'Snackbar can only have 1 action, use Notice if your message require many messages' );
+		warning( true, 'Snackbar can only have 1 action, use Notice if your message require many messages' );
```

Comment:
Out of interest, what does the `true` mean?

Reply:
It's the condition to show the warning: https://github.com/WordPress/gutenberg/tree/master/packages/warning#default

It could be `warning( actions && actions.length > 1, 'Snackbar can only...' )` outside the conditional block, but since it's already there I just put the warning inside it.

Reply:
Ah, ok, thanks. All the calls to it seem to be true in this PR, so the argument seems a bit redundant. Although I'm not a fan of overloaded functions, this does seem like an example in which a version of this function which is just `warning( <string> )` could be useful.